dockers/docker-fpm-quagga/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM docker-config-engine
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4 
+
+COPY \
+debs/libnl-route-3-200_3.2.27-1_amd64.deb debs/libnl-genl-3-200_3.2.27-1_amd64.deb debs/libnl-nf-3-200_3.2.27-1_amd64.deb debs/libhiredis0.13_0.13.3-2_amd64.deb debs/libnl-3-200_3.2.27-1_amd64.deb debs/libnl-cli-3-200_3.2.27-1_amd64.deb debs/libswsscommon_1.0.0_amd64.deb debs/libsairedis_1.0.0_amd64.deb debs/libsaimetadata_1.0.0_amd64.deb debs/libteam5_1.26-1_amd64.deb debs/quagga_0.99.24.1-2.1_amd64.deb debs/swss_1.0.0_amd64.deb debs/
+
+RUN dpkg -i \
+debs/libnl-route-3-200_3.2.27-1_amd64.deb debs/libnl-genl-3-200_3.2.27-1_amd64.deb debs/libnl-nf-3-200_3.2.27-1_amd64.deb debs/libhiredis0.13_0.13.3-2_amd64.deb debs/libnl-3-200_3.2.27-1_amd64.deb debs/libnl-cli-3-200_3.2.27-1_amd64.deb debs/libswsscommon_1.0.0_amd64.deb debs/libsairedis_1.0.0_amd64.deb debs/libsaimetadata_1.0.0_amd64.deb debs/libteam5_1.26-1_amd64.deb debs/quagga_0.99.24.1-2.1_amd64.deb debs/swss_1.0.0_amd64.deb 
+
+## Clean up
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
+RUN rm -rf /debs
+
+COPY ["*.j2", "/usr/share/sonic/templates/"]
+COPY ["start.sh", "config.sh", "/usr/bin/"]
+COPY ["daemons", "/etc/quagga/"]
+
+ENTRYPOINT /usr/bin/config.sh   \
+        && /usr/bin/start.sh    \
+        && /bin/bash
```

Comment:
Remove this file. We could add Dockerfile to .gitignore if Dockerfile.j2 exists.